addons/io_scene_gltf2/__init__.py

```diff
@@ -485,81 +479,9 @@ def execute(self, context):
-                self.draw_material_settings()
-            elif self.ui_tab == 'ANIMATION':
-                self.draw_animation_settings()
-
-        def draw_general_settings(self):
-            col = self.layout.box().column()
-            col.prop(self, 'export_format')
-            if self.export_format == 'GLTF_SEPARATE':
-                col.prop(self, 'export_texture_dir', icon='FILE_FOLDER')
-            col.prop(self, 'export_selected')
-            col.prop(self, 'export_apply')
-            col.prop(self, 'export_yup')
-            col.prop(self, 'export_extras')
-            col.prop(self, 'will_save_settings')
-            col.prop(self, 'export_copyright')
-
-        def draw_mesh_settings(self):
-            col = self.layout.box().column()
-            col.prop(self, 'export_texcoords')
-            col.prop(self, 'export_normals')
-            if self.export_normals:
-                col.prop(self, 'export_tangents')
-            col.prop(self, 'export_colors')
-            col.prop(self, 'export_materials')
-            if self.export_materials:
-                col.prop(self, 'export_image_format')
-
-            # Add Draco compression option only if the DLL could be found.
-            if self.is_draco_available:
-                col.prop(self, 'export_draco_mesh_compression_enable')
-
-                # Display options when Draco compression is enabled.
-                if self.export_draco_mesh_compression_enable:
-                    col.prop(self, 'export_draco_mesh_compression_level')
-                    col.prop(self, 'export_draco_position_quantization')
-                    col.prop(self, 'export_draco_normal_quantization')
-                    col.prop(self, 'export_draco_texcoord_quantization')
-                    col.prop(self, 'export_draco_generic_quantization')
-
-        def draw_object_settings(self):
-            col = self.layout.box().column()
-            col.prop(self, 'export_cameras')
-            col.prop(self, 'export_lights')
-
-        def draw_animation_settings(self):
-            col = self.layout.box().column()
-            col.prop(self, 'export_current_frame')
-            col.prop(self, 'export_animations')
-            if self.export_animations:
-                col.prop(self, 'export_frame_range')
-                col.prop(self, 'export_frame_step')
-                col.prop(self, 'export_force_sampling')
-                if self.export_force_sampling:
-                    col.prop(self, 'export_def_bones')
-            col.prop(self, 'export_skins')
-            if self.export_skins:
-                col.prop(self, 'export_all_influences')
-            col.prop(self, 'export_morph')
-            if self.export_morph:
-                col.prop(self, 'export_morph_normal')
-                if self.export_morph_normal:
-                    col.prop(self, 'export_morph_tangent')
-    else:
-        def draw(self, context):
-            pass
+
+    def draw(self, context):
```

Comment:
Is this empty function still needed?  Might be needed by UI code?

Reply:
Yes, still needed by UI code to get panels working correctly